hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -1130,7 +1130,16 @@ NamespaceDescriptor getNamespaceDescriptor(String name)
       throws NamespaceNotFoundException, IOException;
 
   /**
-   * List available namespace descriptors.
+   * List available namespaces
+   *
+   * @return List of descriptors
```

Comment:
Nit: also change this to "List of namespace names", not descriptors anymore

Reply:
Will fix on commit, thanks